src/SessionSetup/SessionSetupUtils.cpp

```diff
@@ -29,4 +29,22 @@ std::shared_ptr<grpc::Channel> CreateGrpcChannel(uint16_t port) {
   return result;
 }
 
+std::unique_ptr<orbit_client_data::ProcessData> TryToFindProcessData(
+    const std::vector<orbit_grpc_protos::ProcessInfo>& process_list,
+    const std::string process_name_or_path) {
+  std::vector<orbit_grpc_protos::ProcessInfo> process_list_copy = process_list;
```

Comment:
Nit: Since you're making a copy anyway you could also consider taking the process_list by value.